webapp/channels/src/components/admin_console/list_table/elapsed_duration_cell.tsx

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import moment from 'moment';
+import React, {useMemo} from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
+
+import OverlayTrigger from 'components/overlay_trigger';
+import Tooltip from 'components/tooltip';
+
+import Constants from 'utils/constants';
+
+interface Props {
+    date?: number;
+}
+
+export function ElapsedDurationCell(props: Props) {
+    const todaysDate = moment().startOf('day').valueOf();
+
+    const {formatMessage} = useIntl();
+
+    const {elapsedDays, exactPassedInDate} = useMemo(() => {
+        const todayMoment = moment();
+        const passedInDateMoment = moment(props.date);
+
+        // TODO: Use Timestamp component here
+        const elapsedDays = todayMoment.diff(passedInDateMoment, 'days');
+        const exactPassedInDate = passedInDateMoment.format(
+            `MMMM DD, Y [${formatMessage({id: 'admin.console.list.table.exactTime.at', defaultMessage: 'at'})}] hh:mm:ss A`,
+        );
+
+        return {
+            elapsedDays,
+            exactPassedInDate,
+        };
+    }, [props.date, todaysDate]);
+
+    if (!props.date) {
```

Comment:
Nit: can we move this to the top? No use instantiating anything if we're not going to render anything anyways.

Reply:
we cant move this to the top because we have hooks defined here, conditional rendering of hooks is a no-no.